app/poros/cafe_categorizer.rb

```diff
@@ -0,0 +1,56 @@
+class CafeCategorizer
+  class << self
```

Comment:
Went with class methods here instead of creating an object for each cafe in the script